301-jenkins-vmss-blue-green/.ci_skip

```diff
@@ -0,0 +1,2 @@
+* The packer used in this template requires a working service principal to build
```

Comment:
Any reason we can't use MSI?

Reply:
Because it is not supported

https://www.packer.io/docs/builders/azure.html#client_secret

Reply:
To add a bit more, Packer is widely used to create OS images in public cloud (Azure, AWS, Google Cloud Platform, etc), and we have MS developers contributing to the Azure support in Packer. 

However, currently, Packer only supports secret based Service Principal, which prevents us from using MSI as the authentication mechanism in the template.